Runtime/McSubscriber.cs

```diff
@@ -508,6 +553,35 @@ public void RemoveRenderAudioSource(AudioSource source)
         {
             _renderer.RemoveAudioTarget(source);
         }
+        private Layer GetSimulcastlayer(string rid)
```

Comment:
We need to replace this with a function that returns SimulcastInfo, if available, this way, all simulcast info can be retrieved from one place. `encoding Id` is not uniquely identifying